test/test_linearization_checker.py

```diff
@@ -1773,6 +1774,313 @@ def test_blex_map_transitivity_with_duplicate_conc_inames():
 # }}}
 
 
+# {{{ Dependency tests
+
+# {{{ test_add_dependency_v2
+
+def test_add_dependency_v2():
```

Comment:
Add docstring explaining what each test is getting at.

Reply:
done